pom.xml

```diff
@@ -417,6 +417,7 @@
                 <configuration>
                     <source>${java.version}</source>
                     <target>${java.version}</target>
+                    <release>${java.version}</release>
```

Comment:
nit: Maybe we can drop `<source>` and `<target>` now if we don't support java 8?